### ISSUE TITLE
Reproducable segfault (possibly caused by exploration)

### ISSUE THREAD

**Issue by digitCruncher** [CleverRaven/Cataclysm-DDA#41902]
# Describe the bug

Driving north in the current game segfaults, although it does seem to vary slightly in its position.

# Steps To Reproduce

Load the attached save
Speed up to 51 km/hr.
Sometimes a car will spawn in the road. Just drive around it.
Wait. The game will segfault in every test I tried before you pass the turn in the road.

# Expected behaviour

No segfault, and I can turn around the corner to get medicine for my sick friend.
The segfault seems to be related to exploration, as waiting 5 minutes does not cause a segfault.

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.E-3769-g1f55aa9 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    DinoMod [DinoMod],
    No Rail Stations [No_Rail_Stations],
    Bionic Slots [cbm_slots]
]

As an aside, I have 8 GB of RAM.

# Additional context

[crash.log](https://github.com/CleverRaven/Cataclysm-DDA/files/4880969/crash.log)
[debug.log](https://github.com/CleverRaven/Cataclysm-DDA/files/4880970/debug.log)
[Fleischmanns.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/4880973/Fleischmanns.zip)

**Comment by EvgenijM86** [comment]
from crash.log:
```
  #8
    (dbghelp: @0xb70596[cataclysm-tiles.exe+0x770596]), 
    (libbacktrace: ZN7mattack7scienceEP7monster+0xc6@0xb70596),
    (libbacktrace: 0xb70596    [unknown src]:0    [unknown func]),
```

mattack::science() was already fixed by #41871 . Downloaded your safe file, drove north until there was a forest, no crash.

**Comment by digitCruncher** [comment]
You are right. I can also verify this (more accurately, build 10813 fixes this bug). Sorry about this (I hoped to close this before anyone else noticed my mistake)
